tests/test_users.py

```diff
@@ -44,6 +44,28 @@ def test_basic_metadata(standard_graph, session, users, groups, permissions):  #
     assert len(md) == 1, "One metadata item"
 
 
+def test_usertokens(standard_graph, session, users, groups, permissions):  # noqa
+    user = users["zorkian@a.co"]
```

Comment:
Should I be doing something different to ensure I don't mutate shared test state?